style-guide/about-style-guide.md

```diff
@@ -0,0 +1,168 @@
+them or include them here.
+
+## About the "Glossary of Preferred Terms" section
+
+Every project has its own unique terms and buzzwords and this glossary can help
+you record your project’s preferred terms. This word list provides a list of
+specific terms that are either common word choice problems or which are unique
+to your project. As a word of caution, try to minimize jargon and buzzwords.
+Also, avoid creating too many project-specific terms. Where possible, use the
+terms that are standard for your industry to avoid confusion.
+
+This section includes a table to highlight or call out any terms that are unique
+or which conflict with your default style guide. For example, the Google
+Developer's Guide uses "open source" but our project prefers "open-source" when
+used as a modifier, as in "open-source software."
+
+## About the "Topic Types and Templates" section
+
+Depending on the type of project you’re creating, some of the Good Docs
+templates will be more relevant than others. In this section, you can link to
+the templates you have adopted for your project. You can also explain when
+contributors should use these templates.
+
+## About the "General Writing Tips" section
+
+This section is optional. You can link to your favorite general writing tips or
+add some of your own general guidelines.
+
+## About the "Accessible Writing" section
+
+In this section, you can link to your preferred style guides about accessible
+writing or add some of your own general guidelines.
+
+## About the "Inclusive and Bias-Free Writing" section
+
+In this section, you can link to your favorite resources about inclusive and
+bias-free writing or add some of your own general guidelines.
+
+## About the "Using Linters" section
+
+This section is optional. [Linters](https://www.writethedocs.org/guide/tools/testing/#style-guide-checking-and-linting)
+are tools that analyze text content to flag common mistakes such as stylistic
+errors and suspicious constructs. An example of a linter would be a tool like
+Grammarly that processes your text and provides suggestions as you write to
+improve the spelling and grammatical structure of the text.
+
+Linters can be used during the editorial process within an editor and as a
+post-processing test of the content when the document is pushed into a revision
+management system as part of an automated continuous integration (CI) cycle.
+
+If you want to take your style guide to the next level, you could adopt a linter
+to enforce your style guide and empower your content creators to make better
+documentation with less cognitive overhead.
+
+For resources and examples of linters, see:
+
+- [The Atom editor’s Google Linter app](https://atom.io/packages/linter-google-styleguide)
+- A tool like [Vale](https://www.writethedocs.org/guide/tools/testing/#vale)
+  for a more automated approach, including styles such as the
+  [Microsoft Writing Style Guide](https://github.com/errata-ai/styles)
```

Comment:
I expected this link to take me directly to the MS style for Vale. I'd consider reworking this. Perhaps something like:

> A tool like [Vale](https://www.writethedocs.org/guide/tools/testing/#vale), which can automate [several popular styles](https://github.com/errata-ai/styles)

Reply:
That's good feedback! I liked that suggestion and I've adopted it.